src/main.rs

```diff
@@ -15,6 +20,32 @@ fn parse_matches() -> ArgMatches<'static> {
                 .default_value("default")
                 .help("namespace to deploy test cluster"),
         )
+        .arg(
+            Arg::with_name("deploy_method")
```

Comment:
This is entirely up to you, but you could have clap do all the work for you if you break up this arg into three:

```
--local-path <path_to_repo>
--tar
--skip
```

You can make a group of those, so that only one is specified. I'm not sure what `skip` does just, but it might be possible to omit it entirely